xayn-ai/src/ranker/context.rs

```diff
@@ -0,0 +1,117 @@
+use crate::{data::document_data::CoiComponent, ranker::utils::Id};
+
+/// The context used to calculate a document's score.
+/// https://xainag.atlassian.net/wiki/spaces/M2D/pages/770670607/Production+AI+Workflow#3.2-Context-calculations.
```

Comment:
i'm not sure if we should put private links (private in the sense of access permissions, not in the sense of classified) in public code?

Reply:
I agree, but my hope is that at some point it will be publicly accessible 🙏. We also have the issue with the jira tickets